.travis.yml

```diff
@@ -35,37 +35,37 @@ matrix:
     - <<: *linux_clang
       env:
           # For Travis's Ubuntu 14.04, the libstdc++ is broken with clang
-        - MATRIX_EVAL="CXX=clang++-3.8 && CXXFLAGS_OS=-stdlib=libc++"
+        - MATRIX_EVAL="CXX=clang++-3.8 && CXXFLAGS_OS=-stdlib=libc++ -std=c++14"
```

Comment:
I think we use C++1y?

Reply:
Yes, I found that setting c++14 fixes it, if it really will, will then try c++1y.